pkg/apis/apps/types.go

```diff
@@ -871,6 +874,8 @@ type ReplicaSetCondition struct {
 	Type ReplicaSetConditionType
 	// Status of the condition, one of True, False, Unknown.
 	Status api.ConditionStatus
+	// The last time this condition was updated.
+	LastUpdateTime metav1.Time
```

Comment:
is it necessary to guard this by a feature flag and validate its usage by api server?